hack/lib/test.sh

```diff
@@ -76,7 +76,7 @@ kube::test::object_assert() {
   local args=${5:-}
 
   for j in $(seq 1 ${tries}); do
-    res=$(eval kubectl get -a "${kube_flags[@]}" ${args} $object -o go-template=\"$request\")
+    res=$(eval kubectl get "${kube_flags[@]}" ${args} $object -o go-template=\"$request\")
```

Comment:
Can you point me to a place where we don't need it, that this change is required?

Reply:
> Can you point me to a place where we don't need it, that this change is required?

@soltysh When `--label` is specified. This is exactly what this PR is addressing. `--label` and `--all` should not be specified together.

Reply:
What I was asking was, do we have a specific test that uses both (`-l` and `--all`), if so fix that particular test by introducing a new method instead of changing all tests with `-a` which looks ugly and will cause problems.

Reply:
@soltysh Currently `-a` is only applicable for `Pods`. So I removed redundant `-a` for other tests.

PTAL. Thanks.

Reply:
> @soltysh Currently -a is only applicable for Pods. So I removed redundant -a for other tests.
> 
> PTAL. Thanks.

`-a` is `--show-all`, not `--all`.  That makes this change unnecessary, right?